docker/frontend/Dockerfile

```
@@ -1,41 +1,20 @@
-FROM node:alpine
+FROM node:12-alpine as build-stage
+
+WORKDIR /app
+ENV PATH /app/node_modules/.bin:$PATH
+COPY . ./
+RUN npm i
+RUN npm run build
+
+FROM nginx:stable-alpine
+COPY --from=build-stage /app/build/*.html /usr/share/nginx/html/
+COPY --from=build-stage /app/build/*.json /usr/share/nginx/html/2019-2-Track-Frontend-A-Goryakin/
+COPY --from=build-stage /app/build/*.js /usr/share/nginx/html/2019-2-Track-Frontend-A-Goryakin/
+COPY --from=build-stage /app/build/*.png /usr/share/nginx/html/2019-2-Track-Frontend-A-Goryakin/
+COPY --from=build-stage /app/build/*.ico /usr/share/nginx/html/2019-2-Track-Frontend-A-Goryakin/
+COPY --from=build-stage /app/build/*.css /usr/share/nginx/html/2019-2-Track-Frontend-A-Goryakin/
+COPY --from=build-stage /app/build/static /usr/share/nginx/html/2019-2-Track-Frontend-A-Goryakin/static
```

Лучше сразу весь билд скопировть, чем создавать промежуточный слой для копирования одного по одному файлу :)
```suggestion
COPY --from=build-stage /app/build /usr/share/nginx/html/2019-2-Track-Frontend-A-Goryakin
```

---

Попробую сделать копирование нескольких типов файлов в одной инструкции. Если сразу копировать весь билд, то приложение будет доступно по адресу http://localhost/2019-2-Track-Frontend-A-Goryakin/. А в том случае, как сделано сейчас, оно доступно по адресу http://localhost/ :)